src/Card/Card.stories.js

```diff
@@ -0,0 +1,194 @@
+
+import ThemeProvider from "../Theming/ThemeProvider";
+import * as Icons from "../icons";
+import Heading from "../Heading";
+import Text from "../Text";
+
+import Card from "./index";
+
+setAddon(chaptersAddon);
+
+const getIcons = defaultIcon => select("Icon", Object.keys(Icons), defaultIcon);
+const getIcon = source => Icons[source];
+
+const options = {
+  showSource: true,
+  allowSourceToggling: false,
+  showPropTables: false,
+  allowPropTablesToggling: false,
+};
+
+storiesOf("Card", module)
+  .addDecorator(withKnobs)
+  .addDecorator(
+    styles({
+      background: defaultTokens.paletteCloudLight,
+      padding: "20px",
+    }),
+  )
+  .addWithChapters("Default", () => {
+    const title = text("Title", "Card with title");
+    const description = text("Description");
+    return {
+      info: "This is the default configuration of this component.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Card
+                    icon={<Icons.Airplane color="attention" />}
+                    title={title}
+                    description={description}
+                  />
+                </ThemeProvider>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Card with description", () => {
+    const title = text("Title", "Card with title & description");
+    const description = text("Description", "This is a description of the card.");
+    return {
+      info:
+        "Card component is a simple container for grouping some relevant information. It’s possible to add title and description. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Card
+                    icon={<Icons.Airplane color="attention" />}
+                    title={title}
+                    description={description}
+                  />
+                </ThemeProvider>
+              ),
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Card with sections", () => {
+    const title = text("Title", "Card with sections");
+    const description = text("Description", "This is a description of the card.");
+    return {
+      info:
+        "Card sections allow you to create separate sections in every card when you need to create more advanced content structure. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => (
+                <ThemeProvider>
+                  <Card
+                    icon={<Icons.Airplane color="attention" />}
+                    title={title}
+                    description={description}
+                    sectioned
```

Comment:
Can we get rid of this `sectioned` prop?
We can check if children contains `Section` component - something like this:
```js
let sectioned = false;
children.map(child => {
  if (child.type === Section) {
    sectioned = true;
  }
  return sectioned;
});
```

Reply:
Let me know @williamkolmacka